app/helpers/course_queues_helper.rb

```diff
@@ -6,4 +6,8 @@ def serialize_request(request)
       resolver:  { except: User::PROTECTED_FIELDS }
     }})
   end
+
+  def serialize_queue_ids(queue)
+    queue.as_json(only: [:id])
```

Comment:
I think we need to add something like this for the sorting you're doing clientside to work:

```suggestion
    queue.as_json(only: [:id]).map { |entry| entry["id"] }
```

Right now this is returning an array of objects with a key `id`, so the frontend sort doesn't seem to be working correctly.